credoai/reporting/plot_utils.py

```diff
@@ -6,12 +6,62 @@
 
 DEFAULT_COLOR = '#4937c0'
 
-def get_style(rc=None, figsize=6, figure_ratio=1):
-    figsize = [figsize, figsize*figure_ratio]
-    style = {'figure.figsize': figsize,
-             'figure.dpi': 200}
```

Comment:
For some reason, this `dpi` setting is not actually applied in some of the plots. If quickstart demo is run, this for example is some plots are made.
![image](https://user-images.githubusercontent.com/92744575/156010657-6d2ed16a-973f-4049-8a08-a991c6b7abed.png)
![image](https://user-images.githubusercontent.com/92744575/156010780-47a73fe1-b994-4c07-99b0-5d16b7eac7fb.png)

Reply:
I'll look into this.

Reply:
Issue I found, which I have fixed.
* Jupyter defaults to displaying plots as png's.
*  It uses dpi as a scaling factor. So it isn't that easy to create a plot of a specific size and up the resolution.
*  I can make it so the plots are automatically created as svg's. This solves the issues (with some minor overhead for plotting). I've done this by adding the lines to all of our demo notebooks (and later to our reports).